Ical.Net/Serialization/DataTypes/DurationSerializer.cs

```diff
@@ -0,0 +1,108 @@
+﻿//
+// Copyright ical.net project maintainers and contributors.
+// Licensed under the MIT license.
+//
+
+using System;
+using System.IO;
+using System.Text;
+using System.Text.RegularExpressions;
+using Ical.Net.DataTypes;
+
+namespace Ical.Net.Serialization.DataTypes;
+
+public class DurationSerializer : SerializerBase
+{
+    public DurationSerializer() { }
+
+    public DurationSerializer(SerializationContext ctx) : base(ctx) { }
+
+    public override Type TargetType => typeof(Duration);
+
+    public override string SerializeToString(object obj)
```

Comment:
Maybe just
```cs 
public override string SerializeToString(object obj)
    => !(obj is Duration duration) ? null : SerializeToString(duration);
```

Reply:
Certainly, thanks. This class is based on a copy of the previous `TimeSpanSerializer`,  could easily see some leftovers from there.